src/pages/Login/Login.tsx

```diff
@@ -23,10 +26,12 @@ const Login: React.FC = () => {
     initialValues: {
       username: "",
       password: "",
+      rememberMe: false,
     },
     validationSchema: Yup.object({
       username: Yup.string().required("Username is required"),
       password: Yup.string().required("Password is required"),
+      rememberMe: Yup.boolean(),
```

Comment:
Why do we need this  **Yup.boolean(),**? how to use users can able to give only boolean values

Reply:
only Boolean value is allowed, right? then again why we are check weather it is boolean or not